vars/promoteTarArtifacts.groovy

```diff
@@ -16,78 +16,94 @@ void call(Map args = [:]) {
     def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
```

Comment:
The name of this file should change to `promoteArtifacts` or maybe `promoteDistributions`?

Reply:
I think using distribution is inaccurate compares to artifacts.
We are moving only parts of all the distribution artifacts to production bucket.
If this one is named promoteDistribution then docker/yum should also be promoted in this same lib, which is never the case anyway.

I still think promoteAritfacts is better at this point.
Open to suggestion.

Thanks.

Reply:
`promoteAritfacts` works for me, currently it's called ` promoteTarAritfacts`

Reply:
fixed now.